boards/qemu-i386/include/board.h

```diff
@@ -18,8 +18,8 @@
  * @author      René Kijewski <rene.kijewski@fu-berlin.de>
  */
 
-#ifndef __RIOT__BOARDS__QEMU_I386__BOARD__H
-#define __RIOT__BOARDS__QEMU_I386__BOARD__H
+#ifndef RIOT__BOARDS__QEMU_I386__BOARD__H_
+#define RIOT__BOARDS__QEMU_I386__BOARD__H_
```

Comment:
I can remove the double underscores, but I won't rename completely because I fear a potential clash with qemu headers.